package.json

```diff
@@ -19,14 +21,18 @@
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
+    "axe-core": "^4.2.0",
     "babel-eslint": "^10.1.0",
+    "cypress": "^6.9.1",
+    "cypress-axe": "^0.12.2",
     "eslint": "^7.12.1",
+    "eslint-plugin-cypress": "^2.11.2",
```

Comment:
I think we should move the cypress dependencies/install in to `/client`, it won't be needed for the other sub-repos.